src/tools/perf/lib/libperf.c

```diff
@@ -985,27 +984,69 @@ static void ucp_perf_test_err_handler(void *arg, ucp_ep_h ep,
               ucs_status_string(status));
 }
 
-static ucs_status_t ucp_perf_test_receive_remote_data(ucx_perf_context_t *perf)
+static ucs_status_t ucp_perf_test_rkey_pack(ucx_perf_context_t *perf,
```

Comment:
maybe no need to separate ucp_perf_test_rkey_pack/unpack functions now?

Reply:
I prefer to leave it and make the code more readable

Reply:
ok